scripts/sql/xavier-changes-2016_April/aatamsDB_queries_#319.sql

```diff
@@ -0,0 +1,189 @@
+﻿set search_path = aatams, public;
+
+begin;
+
+---- Fix issues listed in Andre's spreadsheet, Sheet 1
+
+--------------------
+-- 1. Delete 'mq' ahd 'Stradbroke Workshop' projects
+DELETE FROM sensor WHERE id IN (77640673, 77640675, 77640701);
+DELETE FROM device WHERE id IN (77640671, 77640699);
+DELETE FROM project_role WHERE id = 129352710;
+DELETE FROM organisation_project WHERE id = 129352708;
+DELETE FROM project WHERE name = 'mq' OR name = 'Stradbroke Workshop';
+
+---- Test results
+-- SELECT * FROM sensor WHERE transmitter_id = 'A180-1105-1111' OR transmitter_id = 'A180-1105-2222' OR transmitter_id = 'A180-1105-3333';
+-- SELECT * FROM project_role WHERE project_id IN (77640666, 129352707);
+-- SELECT * FROM organisation_project WHERE project_id IN (77640666, 129352707);
+-- SELECT * FROM project WHERE name = 'mq' OR name = 'Stradbroke Workshop';
+
+--------------------
+-- 2. Delete deployment 2015-01-19, 10:02:06, Coral Bay BB6, receiver 114574
+DELETE FROM receiver_deployment WHERE id = 129352973;
+
+---- Test results
+-- SELECT receiver_deployment.id, serial_number, initialisationdatetime_timestamp, deploymentdatetime_timestamp, deploymentdatetime_zone 
+-- FROM receiver_deployment
+-- JOIN device ON device.id = receiver_deployment.receiver_id 
+-- WHERE serial_number = '114574';
+-- SELECT COUNT(*) FROM aatams.valid_detection WHERE receiver_deployment_id = 129352973; -- 0 record associated with this deployment
+
+--------------------
+-- 3. Delete deployment CL1, 101848, 23/08/08, 15:30
+DELETE FROM receiver_deployment WHERE id = 4479625;
+
+---- Test results
+-- SELECT receiver_deployment.id, serial_number, initialisationdatetime_timestamp, deploymentdatetime_timestamp, deploymentdatetime_zone 
+-- FROM receiver_deployment
+-- JOIN device ON device.id = receiver_deployment.receiver_id 
+-- WHERE serial_number = '101848';
+-- SELECT COUNT(*) FROM aatams.valid_detection WHERE receiver_deployment_id = 4479625; -- 0 record associated with this deployment
+
+--------------------
+-- 4. Delete deployments NL1, 101764, 22/01/08, 23:30
+DELETE FROM receiver_deployment WHERE id IN (2917441, 2917440);
+
+---- Test results
+SELECT receiver_deployment.id, serial_number, initialisationdatetime_timestamp, deploymentdatetime_timestamp, deploymentdatetime_zone 
```

Comment:
Perhaps this should be commented out as well, so it's distinct from the queries?